tests/UI/pages/BO/BObasePage.js

```diff
@@ -513,6 +514,18 @@ class BOBasePage extends CommonPage {
       );
     }
   }
+
+  /**
+   * Search in BackOffice
+   * @param page {Page} Browser tab
+   * @param query {string} String
+   * @returns {Promise<void>}
+   */
+  async search(page, query) {
+    await this.setValue(page, this.navbarSarchInput, query);
+    await page.keyboard.press('Enter');
+    await page.waitForNavigation('networkidle');
```

Comment:
What does the 'networkidle' value correspond to?

Reply:
`networkidle` is fired when no new network requests are made for 500 ms